pkgs/applications/graphics/krop/default.nix

```diff
@@ -24,6 +24,10 @@ python3Packages.buildPythonApplication rec {
    "\${qtWrapperArgs[@]}"
   ];
 
+  postInstall = ''
+    install -Dt $out/share/applications krop.desktop
```

Comment:
The default behavior of `install` is to make the file executable. Hence:

```diff
diff --git i/pkgs/applications/graphics/krop/default.nix w/pkgs/applications/graphics/krop/default.nix
index 298ba1011e5..2337dfb370e 100644
--- i/pkgs/applications/graphics/krop/default.nix
+++ w/pkgs/applications/graphics/krop/default.nix
@@ -25,7 +25,7 @@ python3Packages.buildPythonApplication rec {
   ];
 
   postInstall = ''
-    install -Dt $out/share/applications krop.desktop
+    install -m666 -Dt $out/share/applications krop.desktop
   '';
 
   # Disable checks because of interference with older Qt versions // xcb
```

Reply:
Done.